stable/mongodb/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mongodb
-version: 7.2.8
+version: 7.2.9
```

Comment:
I think this should be a minor version.

Reply:
Why? My reasoning is that this feature has already existed but didn't work for some particular configuration values, so that is clearly a bug, and this should be a bugfix release.